libafl/src/corpus/testcase.rs

```diff
@@ -93,24 +92,15 @@ where
 {
     /// The [`Input`] of this [`Testcase`], or `None`, if it is not currently in memory
     input: I,
-    /// Time needed to execute the input
-    exec_time: Option<Duration>,
     /// Parent [`CorpusId`], if known
     parent_id: Option<CorpusId>,
     /// If the testcase is "disabled"
     disabled: bool,
     /// has found crash (or timeout) or not
     objectives_found: usize,
-    /// Vector of `Feedback` names that deemed this `Testcase` as corpus worthy
-    #[cfg(feature = "track_hit_feedbacks")]
-    hit_feedbacks: Vec<Cow<'static, str>>,
```

Comment:
We can't remove these, they are needed

Reply:
specifically, the afl-fuzz clone uses them to track what feedbacks hit something

Reply:
Why have an extra struct?

Reply:
For the experiment,
we are using an independent parser program to parse it. but Testcase has the SerdeanyMap which i can't parse. (because it's not registered)
so some parts of the `Testcase` is removed

Reply:
You could create your own deserializer that ignores unregistered types
or just also build the fuzzer into the independent parser (easier)